app/dashboard/models.py

```diff
@@ -97,3 +97,4 @@ class Widget(DashboardBase):
     rows = models.IntegerField()
```

Comment:
Choices for color_pallete need to use TextChoices, which is sublclassed to model and inherits from models.TextChoices. See archive_type in measurement.Archive model. 
Only choice will be 'squac for now.